src/app/fetcher/htmlOnlyFetcher.js

```diff
@@ -24,26 +24,28 @@ export default async function fetch(url) {
     options.agent = new HttpProxyAgent(process.env.HTTP_PROXY);
   }
 
-  try {
-    const response = await nodeFetch(url, options);
-
-    if (!response.ok) {
-      throw new InaccessibleContentError(`Received HTTP code ${response.status} when trying to fetch '${url}'`);
-    }
+  let response;
 
-    const mimeType = response.headers.get('content-type');
-
-    return {
-      mimeType,
-      content: await (mimeType.startsWith('text/') ? response.text() : response.buffer()),
-    };
+  try {
+    response = await nodeFetch(url, options);
```

Comment:
as we are using a user agent on puppeteer, maybe add it it here also

Reply:
Why? We do not have issues with the `htmlOnlyFetcher`?
In any case, if we want to do that, we will do it in another PR.